kotlintest-runner/kotlintest-runner-jvm/src/main/kotlin/io/kotlintest/runner/jvm/NamedThreadFactory.kt

```diff
@@ -0,0 +1,11 @@
+package io.kotlintest.runner.jvm
+
+import java.util.concurrent.ThreadFactory
+import java.util.concurrent.atomic.AtomicInteger
+
+internal class NamedThreadFactory(val name: String) : ThreadFactory {
```

Comment:
Now that it's moved to it's own file, what about creating a test for this guy? I'm unsure if it's possible due to that thing you said about testing in another module.

But maybe a test for him would be nice. It would catch the same name bug

Reply:
Test added